public/lib/column-defaults.js

```diff
@@ -258,8 +258,8 @@ const columnDefaults = [{
     sortField: 'printWordCount'
 },{
     name: 'publicationlocation',
```

Comment:
Could we change the `name` field too? And also rename the html template above? Having them called `publicationLocation` but displaying them as `Print location` will very soon become confusing.

Happy to pair on this.

Reply:
Done.